index.html

```diff
@@ -64,5 +64,159 @@ <h1 class="headline-alexandra_small">
         </ul>
       </div>
     </section>
+    <section class="works">
+      <div class="tonic">
+        <div><img src="images/Snapshoot Portfolio.png" alt="" /></div>
+        <div class="right">
+          <h1>Tonic</h1>
+          <div class="back">
```

Comment:
- [ ] Please, It would be better for all the titles except the main that indicates the web to use another ```hx```  like ```h2, h3, h4, ...``` to replace the ```<h1>``` as we only have to use a single ```h1``` on the page title to indicate what the entire web page is about 😎😎

**kindly fix this issue on every project name 🙏**